### PR TITLE
Allow react-webcam to receive custom style

### PR DESCRIPTION
**Pull request by stefaniacardenas**
Allow react-webcam to receive custom style in order to easily fix UI issues.

### CODE REVIEW

**Comment by rfreitas** [comment]
won't class name do the trick? is it because we cannot override the `transform` in the inlined style? because inline style should be avoided when possible.

**Comment by stefaniacardenas** [comment]
Not in this case. I'm calculating some css values in my js file and I'm not sure how to pass those values to the stylesheet. It seems easier to me to add them inline.

File: src/react-webcam.js [hunk 1]
```diff
@@ -220,7 +221,8 @@ export default class Webcam extends Component<CameraType, State> {
         style={{
           // not necessary to add prefixes, since all browsers that support camera
           // support transform
-          transform: this.state.mirrored ? 'scaleX(-1)' : ''
+          transform: this.state.mirrored ? 'scaleX(-1)' : '',
+          ...this.props.style
```
Comment (elhesuu) [inline src/react-webcam.js]: nit: trailing comma?

File: src/react-webcam.js [hunk 2]
```diff
@@ -31,7 +31,8 @@ type CameraType = {
     'image/png' |
     'image/jpeg'
   ,
-  className?: String
+  className?: String,
+  style: Object
```
Comment (elhesuu) [inline src/react-webcam.js]: trailing comma here too?

**Pull request merged**